src/main/java/io/github/nataliqa/model/CatalogItem.java

```diff
@@ -0,0 +1,17 @@
+package io.github.nataliqa.model;
+
+public class CatalogItem {
```

Comment:
Check the necessity of CatalogItem class. Objects of this class are not used in your project.